woocommerce/woocommerce-android
# Promote extensions

⚠️ This PR depends o this FluxC PR https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2909

Part of: #9550 

### Why
As part of the product extension project, this PR adds an "Explore button" to the product types filter whose correspondent extension is not installed on the store.

This button will redirect the user to the extension's webpage with the idea of promoting it.

### Description
This PR adds the `explore` option to the product filter. With the changes introduced here, the product filter checks for the installed Woo extensions on the merchant's site, and it highlights the missing extensions (woo product extensions supported that are not installed)

### Testing instructions
1. Open the products tab
2. Tap on Filter->Product type
3. Check that Woo extensions supported on the filter (Subscriptions, Product Bundles, and Composite products) when installed, can be selected and, when not, display a button with the text `Explore!`.
4. Tap on the Explore! filter options and check that it redirects to the extension page selected.


### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/18119390/7c7d155b-2820-4705-97b0-eb0db11373e2
